packages/terra-data-grid/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 * Added
   * Added ability to select a range of cells to FlowsheetDataGrid.
 
+* Changed
+  * Updated FlowsheetDataGrid to no longer have column headers.
```

Comment:
I think this will now go under the Added section.  Also, you are allowing the column headers to be hidden.  However, it is still possible.  Maybe this update was missed based on your other updates in the PR.